app/src/main/java/org/oppia/android/app/testing/LessonThumbnailImageViewTestActivity.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import org.oppia.android.R
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+class LessonThumbnailImageViewTestActivity : InjectableAppCompatActivity() {
```

Comment:
Please add a brief KDoc here explaining what this is (even though it's sort of self-explanatory), e.g.:

```
/** Test-only activity for verifying behaviors of LessonThumbnailImageView. */
```

Reply:
Done